packages/monitoring/src/v3/index.js

```diff
@@ -15,21 +15,25 @@
  */
 'use strict';
 
-var groupServiceApi = require('./group_service_api');
-var metricServiceApi = require('./metric_service_api');
+var groupServiceClient = require('./group_service_client');
+var metricServiceClient = require('./metric_service_client');
 var gax = require('google-gax');
 var extend = require('extend');
+var union = require('lodash.union');
 
 function v3(options) {
   options = extend({
     scopes: v3.ALL_SCOPES
   }, options);
   var gaxGrpc = gax.grpc(options);
   var result = {};
-  extend(result, groupServiceApi(gaxGrpc));
-  extend(result, metricServiceApi(gaxGrpc));
+  extend(result, groupServiceClient(gaxGrpc));
+  extend(result, metricServiceClient(gaxGrpc));
   return result;
 }
-v3.SERVICE_ADDRESS = groupServiceApi.SERVICE_ADDRESS;
-v3.ALL_SCOPES = [];
+v3.SERVICE_ADDRESS = groupServiceClient.SERVICE_ADDRESS;
+v3.ALL_SCOPES = union(
```

Comment:
This has occurred a couple of other times. `union` is an unnecessary dependency, as `ALL_SCOPES` are the same between the services.